src/AppBundle/Tests/Controller/SubstituteControllerTest.php

```diff
@@ -41,100 +58,69 @@ public function testShow()
 
         // Assert that we have the correct page
         $this->assertEquals(1, $crawler->filter('h1:contains("Vikarer")')->count());
-        $this->assertEquals(1, $crawler->filter('td:contains("Marius")')->count());
-        $this->assertEquals(1, $crawler->filter('td:contains("Svendsen")')->count());
+        $this->assertEquals(1, $crawler->filter('td:contains("Markus")')->count());
+        $this->assertEquals(1, $crawler->filter('td:contains("Gundersen")')->count());
 
-        // Assert that we don't have the edit/delete buttons as team (1 delete button is the hidden modal button, so we have to check = 1)
-        $this->assertEquals(1, $crawler->filter('a:contains("Slett")')->count());
+        // Assert that we don't have the edit/delete buttons as team
+        $this->assertEquals(0, $crawler->filter('a:contains("Slett")')->count());
         $this->assertEquals(0, $crawler->filter('a:contains("Rediger")')->count());
-
-        // Assistant user
-        $client = static::createClient(array(), array(
-            'PHP_AUTH_USER' => 'assistent',
-            'PHP_AUTH_PW' => '1234',
-        ));
-
-        $client->request('GET', '/kontrollpanel/vikar');
-
-        // Assert that the page response status code is 403 access denied
-        $this->assertEquals(403, $client->getResponse()->getStatusCode());
     }
 
     public function testShowSubstitutesByDepartment()
     {
-        // Admin user
-        $client = static::createClient(array(), array(
-            'PHP_AUTH_USER' => 'admin',
-            'PHP_AUTH_PW' => '1234',
-        ));
+        // Team leader
+        $client = self::createTeamLeaderClient();
 
-        $crawler = $client->request('GET', '/kontrollpanel/vikar/1');
+        $crawler = $client->request('GET', '/kontrollpanel/vikar/semester/2');
```

Comment:
```php
$client = self::createTeamLeaderClient();
$crawler = $client->request('GET', '/kontrollpanel/vikar/semester/2');
$this->assertEquals(200, $client->getResponse()->getStatusCode());
```
kan erstattes med:
```php
$crawler = $this->teamLeaderGoTo('/kontrollpanel/vikar/semester/2');
```
Se beskrivelsen på https://github.com/vektorprogrammet/vektorprogrammet/pull/508 for mer info